purchases/src/main/kotlin/com/revenuecat/purchases/BillingWrapper.kt

```diff
@@ -179,11 +179,13 @@ internal class BillingWrapper internal constructor(
         executeRequestOnUIThread {
             val params = BillingFlowParams.newBuilder()
                 .setSkuDetails(skuDetails)
-                .setAccountId(appUserID)
-                .setOldSku(upgradeInfo?.oldSku)
+                .setObfuscatedAccountId(appUserID.sha256())
```

Comment:
I can't test this because the `params` doesn't have a `getObfuscatedAccountId`

Reply:
I guess the problem there is that we have an implicit dependency on `BillingFlowParams`, since we're using a static method. 
It's annoying as hell to test, but it's not entirely impossible. Just might not be worth it. 

Thoughts: 
- You can have `BillingFlowParams` as a variable, i.e.: `billingFlowParamsClass`, which you would then use DI to override with a custom class for testing. The custom class could inherit from `BillingFlowParams` and override `setObfuscatedAccountId` so that it stores the param sent in before going forward. 
- You could do the same thing but with the `setObfuscatedAccountId`, with a higher-order function and making sure that it gets called with the right params.  

But again, I'm not sure that any of that it worth the trouble. I think this is fine without going deeper down the rabbit hole.